src/Styleguide/Components/MarketInsights.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import { Responsive } from "../Utils/Responsive"
+import { BorderBox } from "../Elements/Box"
+import { Sans } from "@artsy/palette"
+import { themeGet } from "styled-system"
+import styled from "styled-components"
+
+const TextWrap = styled.div`
+  display: block;
+
+  @media ${themeGet("mediaQueries.xs")} {
```

Comment:
I worry when I see code like this that it won't work in react native (which doesn't support media queries)